README.md

```diff
@@ -29,6 +29,11 @@ Finally, when you finish shut it down:
 docker compose stop
 ```
 
+```bash
+# Deploying assets to a bucket
+aws s3 sync ./out/ s3://my-assets-bucket --delete
```

Comment:
this command deploys the static assets to the infrastructure launched from the terraform folder